package.json

```diff
@@ -3,23 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
+    "babel-jest": "^18.0.0",
     "axios": "^0.15.3",
     "babel-loader": "^6.2.10",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "bootstrap": "^3.3.7",
+    "jest": "^18.1.0",
```

Comment:
We don't need to specify Jest because it's a dependency of react-scripts (which is itself a dependency of create-react-app)